read_buffer/src/column/boolean.rs

```diff
@@ -15,6 +15,15 @@ impl BooleanEncoding {
         }
     }
 
+    /// The estimated total size in bytes of the underlying bool values in the
+    /// column if they were stored contiguously and uncompressed. `include_nulls`
+    /// will effectively size each NULL value as 1b if `true`.
```

Comment:
I find the comment confusing.

This is what I understood, rephrased:

```suggestion
    /// The estimated total size in bytes of the underlying bool values in the
    /// column if they were stored contiguously and uncompressed. 
    /// If `include_nulls` is false, NULL values will be excluded from the count; otherwise
    /// NULL values will still occupy the space of a slot of the underlying type, in bool's case 1b.
```

But still the null takes space in the valid bitmask, do we account for that somewhere else?

Reply:
> But still the null takes space in the valid bitmask, do we account for that somewhere else?
yes that would be accounted for in the `size()` result.